test/framework/proxy.go

```diff
@@ -14,6 +14,9 @@ const (
 	cloudstackHttpProxyVar  = "T_HTTP_PROXY_CLOUDSTACK"
 	cloudstackHttpsProxyVar = "T_HTTPS_PROXY_CLOUDSTACK"
 	cloudstackNoProxyVar    = "T_NO_PROXY_CLOUDSTACK"
+	TinkerbellHttpProxyVar  = "T_HTTP_PROXY_TINKERBELL"
```

Comment:
Use upper-case for acronyms. `TinkerbellHTTPProxyVar`.

Reply:
Thanks for pointing out. I have removed e2e test related files from this PR and will create another PR for that.